robocrys/condense/molecule.py

```diff
@@ -0,0 +1,139 @@
+"""
+This module implements a class to match molecule graphs to molecule names.
+
+Some functionality relies on having a working internet connection.
+"""
+
+import warnings
+from typing import Optional, Tuple
+
+from pkg_resources import resource_filename
+from pubchempy import get_compounds, BadRequestError
+
+from pymatgen import loadfn
+from pymatgen.analysis.graphs import MoleculeGraph
+from pymatgen.io.babel import BabelMolAdaptor
+
+
+class MoleculeNamer(object):
+    name_sources = ('traditional', 'iupac')
+
+    def __init__(self, use_online_pubchem: bool = True,
```

Comment:
slightly nervous about default=True

just don't want the pubchem site changing / going down to bring down all of robocrystallographer, e.g. in the middle of building 100,000 compounds or something